### PR TITLE
Leaderboards

### PR DESCRIPTION
**Pull request by aXises**
This PR outlines the first screen we can see we are initialising the ARP spoof to mimic the router - mimic the Raspberry Pi as the router and then sending packets back to the server and back to the victim. And then we are setting up the drift net and the URL snarf to capture the packets and display them in a readable format. As we can see the victim is browsing the BBC website homepage and then the URL snarf is capturing all the information about the packets including the website they're visiting, the browser, the time, and other useful information. So here we can see the information being recorded by the ARP- URL snarf as well as the images being detected by the drift net.
As we can see the second part of the project is here where we have to input the hash and the hash type and then it retrieves the information from a large database through an API and displays it to the screen and then we can verify it by using a MD5 hasher.

Also adds an unfinished leaderboards table.

### CODE REVIEW

**Review by Assasindie — APPROVED** [review]
i do not understand javascript looks good

**Comment by aXises** [comment]
Please see https://github.com/BradF-99/RepostDetector/pull/20/files#diff-e5ae3f565d1ff9951f7c12439e47bc160cb036464dd6e152f82c279c46397e97 for API interface thank u

**Comment by aXises** [comment]
@BradF-99 please enable GitHub pages to serve from the gh-pages branch

**Comment by BradF-99** [comment]
could this be hosted from the api server directly

**Comment by aXises** [comment]
@BradF-99 yes run `yarn build` then `yarn start` and it will serve on localhost:3000

File: static/src/components/Jzlmao.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+const jzBarrowsRun = "9".repeat(10000);
```
Comment (BradF-99) [inline static/src/components/Jzlmao.tsx:3]: is this true @Josh-Zhou

**Comment by BradF-99** [comment]
> 
> 
> @BradF-99 please enable GitHub pages to serve from the gh-pages branch

its being hosted from https://bradfuller.xyz/RepostDetector/ apparently

File: static/src/components/Jzlmao.tsx (hunk 1 above)
Comment (aXises) [reply static/src/components/Jzlmao.tsx:3]: Can confirm,

signed jz

**Pull request merged**